bento-compose/src/main/java/com/yelp/android/bento/compose/ComposeViewHolder.kt

```diff
@@ -0,0 +1,52 @@
+package com.yelp.android.bento.compose
+
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.platform.ViewCompositionStrategy
+import com.yelp.android.bento.core.ComponentViewHolder
+
+/**
+ * ViewHolder which allows compatability with Jetpack Compose. Basically, this lets you write the
+ * view holders view code with Compose.
+ */
+abstract class ComposeViewHolder<P, T> : ComponentViewHolder<P, T>() {
+
+    private var state: MutableState<T?> = mutableStateOf(null)
+    private var composeView: ComposeView? = null
+    var presenter: P? = null
+    var element: T? = null
+
+    final override fun inflate(parent: ViewGroup): View {
+        return ComposeView(parent.context).apply {
+            setViewCompositionStrategy(
+                ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed
+            )
+            setContent {
+                state = remember { mutableStateOf(element) }
+                presenter?.let { element?.let { nonNullElement -> BindView(it, nonNullElement) } }
```

Comment:
In AutoMviComposeViewHolder we can use rememberd state as well for the better performance :)
(https://sourcegraph.yelpcorp.com/android/auto-mvi/-/blob/auto-mvi-bento/src/main/java/com/yelp/android/automvibento/AutoMviComposeViewHolder.kt#L35:22)